script/utils/InvestableWorkstreams/01-DeployInvestableWorkstream.s.sol

```diff
@@ -197,8 +197,8 @@ contract SetupInvestableWorkstream is Test, DeploymentScript {
         // Now we need to find the MilestoneManager. ModuleManager has a function called `listModules` that returns a list of
         // active modules, let's use that to get the address of the MilestoneManager.
 
-        // TODO: Ideally this would be substituted by a check that that all mandatory modules implement their corresponding interfaces + the same for MilestoneManager
-
+        //::@todo: Ideally this would be substituted by a check that that all mandatory modules implement their corresponding interfaces + the same for MilestoneManager
+        // @0xNuggan are these Contracts still up to date? Can we maybe remove them?
```

Comment:
Same as with SetupToyOrchestrator

Reply:
So if you created an issue for it can I remove the comment then?